spec/integration/manifest.spec.yml

```diff
@@ -562,6 +562,10 @@ spec:
               type: array
               items:
                 type: string
+    allowed_in_fedramp_high:
+      type: boolean
+      description: A flag which indicates if this integration is allowed in our ECH FedRAMP High product offering.
```

Comment:
Could we provide some guidance on what are the requirements for an integration to be allowed in FedRAMP?

Reply:
it would likely just be a link to some external URL describing the (very) complex requirements. is that in scope for package spec?

Reply:
I wouldn't reference FEDRAMP at all here, just FIPS.

Reply:
Agent itself doesn't do anything in service of FEDRAMP besides being able to use FIPS cryptography. How agent is deployed affects whether it meets the FEDRAMP requirements and integrations have no control over that.

Reply:
> it would likely just be a link to some external URL describing the (very) complex requirements. is that in scope for package spec?

I mean a quick guidance, so it helps package developers to decide if they should use this variable or not.

Something like:
```suggestion
      description: Indicate if this package is capable of satisfying FIPS requirements. Set to false if it uses any input that cannot be configured to use FIPS cryptography.
```

Reply:
i think there's two ways of looking at it - describe what's going on with the package itself, which is what your comment does above, or describe how the package can be used in Elastic products, which was my original intention. since the package isn't just used in ECH and talking about FIPS is more informative, i think the former is better so will accept the change.